metecho/api/migrations/0077_add_org_config_name_to_task.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.6 on 2020-05-28 19:49
+
+import sfdo_template_helpers.fields.string
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0076_merge_20200528_1808"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="task",
+            name="org_config_name",
+            field=sfdo_template_helpers.fields.string.StringField(blank=True),
```

Comment:
@wlonk For the migration only, this should get a default value of `dev` for every Task.